### ISSUE TITLE
0.4: ArgumentError: function serve does not accept keyword arguments

### ISSUE THREAD

**Issue by dlfivefifty** [JunoLab/atom-julia-client#195]
```
ERROR: LoadError: ArgumentError: function serve does not accept keyword arguments
while loading /Users/solver/Projects/julia-client/script/boot.jl, in expression starting on line 35
Julia has stopped: 1, null
```

**Comment by MikeInnes** [comment]
I just pushed a fix for this to the julia-0.4 branch, if you pull you should be ok.

**Comment by dlfivefifty** [comment]
Thanks!

**Comment by daftmath** [comment]
Did this fix get applied to 0.5.0?

**Comment by MikeInnes** [comment]
Yes, if you're up to date then you shouldn't see this error on either julia 0.4 or 0.5.
